### ISSUE TITLE
`&` (overlay) operator does not SVG render correctly.

### ISSUE THREAD

**Issue by Fordi** [paulrosen/abcjs#240]
If I understand it correctly, the `&` operator is meant to overlay another voice within a measure, by resetting the cursor to the start of the measure.

Input:

```
X: 1
K: C
| c d e f g  a  &\
  A A A A A  A |
```

Expected (example from EasyABC):

![image](https://user-images.githubusercontent.com/116676/61176243-8ad89e00-a58b-11e9-84e3-b67e1c671074.png)

Actual:

![image](https://user-images.githubusercontent.com/116676/61176252-b491c500-a58b-11e9-950c-b877793fa175.png)

I'll be digging into the sources to see if I can find the "why", but if you nab this before me, I won't be put out.

**Comment by Fordi** [comment]
Looks like it may be the parser:
```
const book = ABCJS.parseOnly(`X: 1\nT: Sample\nK: C\n| c d e f g a b c & A G B F c E d A |`);
book[0].lines[0].staff[0].voices
  .map((voice, vi) => (
    `Voice ${vi}\n\t${voice.map((node, ni) => (
      `Node ${ni}: ${JSON.stringify(node)}`
    )).join('\n\t')}`
)).join('\n');
```

Output:

```
Voice 0
	Node 0: {"type":"bar_thin","el_type":"bar","startChar":20,"endChar":21}
	Node 1: {"pitches":[{"pitch":7,"verticalPos":7}],"duration":0.125,"el_type":"note","startChar":21,"endChar":24}
	Node 2: {"pitches":[{"pitch":8,"verticalPos":8}],"duration":0.125,"el_type":"note","startChar":24,"endChar":26}
	Node 3: {"pitches":[{"pitch":9,"verticalPos":9}],"duration":0.125,"el_type":"note","startChar":26,"endChar":28}
	Node 4: {"pitches":[{"pitch":10,"verticalPos":10}],"duration":0.125,"el_type":"note","startChar":28,"endChar":30}
	Node 5: {"pitches":[{"pitch":11,"verticalPos":11}],"duration":0.125,"el_type":"note","startChar":30,"endChar":32}
	Node 6: {"pitches":[{"pitch":12,"verticalPos":12}],"duration":0.125,"el_type":"note","startChar":32,"endChar":34}
	Node 7: {"pitches":[{"pitch":13,"verticalPos":13}],"duration":0.125,"el_type":"note","startChar":34,"endChar":36}
	Node 8: {"pitches":[{"pitch":7,"verticalPos":7}],"duration":0.125,"el_type":"note","startChar":36,"endChar":38}
	Node 9: {"type":"bar_thin","el_type":"bar","startChar":56,"endChar":57}
Voice 1
	Node 0: {"el_type":"note","duration":0,"rest":{"type":"invisible"},"startChar":20,"endChar":21}
	Node 1: {"type":"bar_thin","el_type":"bar","startChar":20,"endChar":21}
	Node 2: {"pitches":[{"pitch":5,"verticalPos":5}],"duration":0.125,"el_type":"note","startChar":39,"endChar":42}
	Node 3: {"pitches":[{"pitch":4,"verticalPos":4}],"duration":0.125,"el_type":"note","startChar":42,"endChar":44}
	Node 4: {"pitches":[{"pitch":6,"verticalPos":6}],"duration":0.125,"el_type":"note","startChar":44,"endChar":46}
	Node 5: {"pitches":[{"pitch":3,"verticalPos":3}],"duration":0.125,"el_type":"note","startChar":46,"endChar":48}
	Node 6: {"pitches":[{"pitch":7,"verticalPos":7}],"duration":0.125,"el_type":"note","startChar":48,"endChar":50}
	Node 7: {"pitches":[{"pitch":2,"verticalPos":2}],"duration":0.125,"el_type":"note","startChar":50,"endChar":52}
	Node 8: {"pitches":[{"pitch":8,"verticalPos":8}],"duration":0.125,"el_type":"note","startChar":52,"endChar":54}
	Node 9: {"pitches":[{"pitch":5,"verticalPos":5}],"duration":0.125,"el_type":"note","startChar":54,"endChar":56}
	Node 10: {"type":"bar_thin","el_type":"bar","startChar":56,"endChar":57}
```

It looks like Voice 1 (after the `&`) is getting an extra, hidden rest, which may be pushing it forward by one.

**Comment by Fordi** [comment]
Looking at the SVG output, the notes are grouped together as one would expect (e.g., voice 1 rendered before voice 2), voice 2 is just shifted to the right by 2 note widths.

**Comment by Fordi** [comment]
Interestingly, prepending a hidden rest (`x`) before the measure line appears to resolve the problem:

Input:

```
X: 1
K: C
x\
| c d e f g  a  &\
  A A A A A  A |
```

Output: 

![image](https://user-images.githubusercontent.com/116676/61176628-1e14d200-a592-11e9-8cab-476e92504f74.png)

That's as expected, and may indicate that there's some uninitialized counter or other in the SVG renderer that's the culprit.

**Comment by Fordi** [comment]
Hmm... looks like there may be something wrong with getLeftEdgeOfStaff.  Also, checking with different meters and default note lengths, it looks like it's always shifted to the right by 1/4 note.  Just another clue, of course.

**Comment by Fordi** [comment]
Huh.  Seen in 5.6.5, but fixed in 5.7.0.  Closing.
